scripts/feeds/feed_download.py

```diff
@@ -1,6 +1,5 @@
-#!/usr/bin/env python3
+#!/usr/local/bin/python3.6
```

Comment:
Propose to either keep `#!/usr/bin/env python3` or use `#!/usr/bin/env python3.6` in all scripts.